### PR TITLE
Removed dependency to dropwizard-core 

### PR DESCRIPTION
**Pull request by elhoo**
Removed dependencies to dropwizard-core to reduce the number of transitive dependencies.

### CODE REVIEW

File: dropwizard-metrics-influxdb/pom.xml [hunk 1]
```diff
@@ -38,13 +38,14 @@
...
+            <artifactId>dropwizard-metrics</artifactId>
             <version>${dropwizard.version}</version>
         </dependency>
         <dependency>
-            <groupId>io.dropwizard</groupId>
-            <artifactId>dropwizard-validation</artifactId>
-            <version>${dropwizard.version}</version>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>3.4</version>
```
Comment (xiaodong-izettle) [inline dropwizard-metrics-influxdb/pom.xml]: Minor, is it OK to extract this version number, and use a property here? :)

Comment (elhoo) [inline dropwizard-metrics-influxdb/pom.xml]: Actually, my first version had one property there - but all other minor dependencies didn't use properties so I removed it...

But I can add properties for all dependencies.

**Comment by ngong** [comment]
Dependencies cleaned up and dropwizard version updated to 1.0 in PR #55

File: dropwizard-metrics-influxdb/pom.xml (hunk 1 above)
Comment (xiaodong-izettle) [inline dropwizard-metrics-influxdb/pom.xml]: Ah, sorry, did not notice that... :-)

**Comment by xiaodong-izettle** [comment]
Hehe @elhoo, @ngong is faster... :-)

**Comment by xiaodong-izettle** [comment]
@elhoo, we'll merge the content of this one and #55... :)

**Pull request closed without merging**